packaging/scripts/publish_release.sh

```
@@ -0,0 +1,117 @@
+require_env AWS_SECRET_ACCESS_KEY
+
+DOWNLOAD_PREFIX="${DOWNLOAD_PREFIX:-download}"
+CDN_BASE_URL="${CDN_BASE_URL:-https://cdn.aishell.ai}"
+VERSION="${VERSION#v}"
+ARTIFACT_ROOT="${ARTIFACT_ROOT%/}"
+VALIDATION_RETRY_COUNT="${VALIDATION_RETRY_COUNT:-10}"
+VALIDATION_RETRY_DELAY_SECS="${VALIDATION_RETRY_DELAY_SECS:-2}"
+
+mapfile -t ARTIFACT_FILES < <(
+  find "$ARTIFACT_ROOT" -type f \(
+    -name "aish-${VERSION}-linux-*.tar.gz" -o
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)